src/main/java/orlov/programming/spring_core_gym/storage/impl/TrainerStorage.java

```diff
@@ -0,0 +1,73 @@
+package orlov.programming.spring_core_gym.storage.impl;
+
+import lombok.Setter;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.stereotype.Component;
+import orlov.programming.spring_core_gym.model.user.Trainer;
+import orlov.programming.spring_core_gym.storage.Storage;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.HashMap;
+import java.util.Map;
+
+@Log4j2
+@Component
+public class TrainerStorage implements Storage<Long, Trainer> {
```

Comment:
it seems that common logic can be extracted into abstract class, not to duplicate logic